### DISCUSSION TITLE
Symbols code editor

### DISCUSSION THREAD

**Discussion by niero369** [ScratchAddons/ScratchAddons#6122] [Ideas]
Suggestion:
I'm a part of an organization that teaches STEM to kids and we love using Scratch to teach coding! One barrier that we found to come up a lot (especially amongst younger and dyslexic students) is that transitioning from something like ScratchJr to Scratch is challenging and intimidating due to the need for reading as opposed to the easy-to-understand symbols. I think it would be incredible if there was a way to make using Scratch more usable for young children by having some sort of retexture to the blocks that replaces words with symbols, would you folks at Scratch Addons be willing to attempt this?

**Comment by WorldLanguages** [comment]
@niero369 Do you know if there's any suggestion on the Scratch Forums (https://scratch.mit.edu/discuss/1/) we could take as a reference? I wonder if someone has already thought of the icons already. That's probably the most challenging part.

**Comment by Samq64** [comment]
Related: #4039
